sdk/spring/scripts/replace_util.py

```diff
@@ -105,72 +107,91 @@ def replace(module):
     """
     Replace action
     :param module: module name
-    :return:
     """
     for pom in config[module].keys():
         replace_artifact_id(module, pom)
         replace_version(module, pom)
 
 
+def get_str(tuple_obj):
+    """
+    Return str list for tuple obj for logger.
+    :param tuple_obj: tuple obj
+    :return: string list
+    """
+    str_list = list()
+    for item in tuple_obj:
+        str_list.append(str(item))
+    return str_list
+
+
 def replace_artifact_id(module, pom):
     """
-    Replace artifactId in dependency
+    Replace artifactId in dependency and plugin part.
     :param module: module name
     :param pom: pom file path
-    :return:
     """
     log.debug('Replacing artifact id in file: {}'.format(pom, module))
-    artifact_id_pairs = config[module][pom]['artifact_id_pairs']
-    log.debug('Module: {}, artifact ids: {}'.format(module, artifact_id_pairs))
+    pom_dict = config[module][pom]
+    if not pom_dict.__contains__(ARTIFACT_ID_PAIRS_KEY_NAME):
```

Comment:
Please use `in` instead of `__contains__`.
Refs: https://stackoverflow.com/questions/5312778/how-to-test-if-a-dictionary-contains-a-specific-key